### PR TITLE
Prepare for v4.1.7 release

### PR DESCRIPTION
**Pull request by sebrandon1**
https://github.com/test-network-function/cnf-certification-test-partner/releases/tag/v4.1.7

### CODE REVIEW

**Review by rdavid — APPROVED** [review]

**Review by ramperher — CHANGES_REQUESTED** [review]
Please do not merge this until having a green DCI job, I'll start to check why it's failing since yesterday

**Comment by sebrandon1** [comment]
Sounds good!

**Comment by ramperher** [comment]
Reviewed with @sebrandon1 , I think we need the patch for `lifecycle-no-pvs-on-localstorage` test name introduced in https://github.com/test-network-function/cnf-certification-test/pull/937, to be renamed again to its correct name, `lifecycle-storage-required-pods`, before merging this one, which implies a stable version.

**Pull request merged**
